workflow/rules/common.smk

```diff
@@ -82,7 +82,7 @@ def get_fastqs(wildcards):
 
 
 def get_resource(name):
-    return str((Path(workflow.snakefile).parent.parent / "resources") / name)
+    return str((Path(workflow.snakefile).parent.parent.parent / "resources") / name)
```

Comment:
Why do need to add another `.parent` here?

Reply:
Ah, I had this too. Needed to add it for the run on the actual machines but I had to exclude it again for git actions. Any ideas where this suddenly comes from?

Reply:
Yes, an API change in Snakemake. 6.3 provides the path to the current snakefile instead of the main snakefile.

Reply:
That's why the workflow now also fails if snakemake version is not at least 6.3